src/services/session/session.service.ts

```diff
@@ -6,10 +6,19 @@ import { ISignInInfo } from 'ch-node-session-handler/lib/session/model/SessionIn
 import { Request } from 'express'
 import { provide } from 'inversify-binding-decorators'
 
+import { DissolutionSession } from 'app/models/dissolutionSession'
+import Optional from 'app/models/optional'
+
 @provide(SessionService)
 export default class SessionService {
 
   public getAccessToken(req: Request): string {
     return req.session!.get<ISignInInfo>(SessionKey.SignInInfo)!.access_token!.access_token!
   }
+  public getDissolutionSession(req: Request): Optional<DissolutionSession> {
+    return req.session!.getExtraData<DissolutionSession>('dissolution')
```

Comment:
I don't think you need a SessionKey enum, but just make this a constant at the top of the file:
```
private readonly DISSOLUTION_SESSION_KEY: string = 'dissolution'
```